test/browserHashes.spec.js

```diff
@@ -43,7 +43,11 @@ describe('Browser hash implementations', function() {
                         if (truncate) {
                             digest = digest.slice(0, truncate);
                         }
-                        expect(digest.toString('hex')).to.equal(expected);
+                        //in node <= 0.10 digest sometimes returns a Dataview, should be buffer.
```

Comment:
Are these tests running in node.js? This should only be running in browser environments, and is using the 3rd party `Buffer` package instead of node.js' `Buffer` package. We can place browser-specific tests in a separate folder and exclude them from being run by mocha in the npm unit script.

Reply:
Ah right! The change mean to fix the test. But actually we can remove it out of node test. I didn't realize this is browser-only although the name already told me